eval.py

```diff
@@ -75,8 +86,22 @@
 # =================================================================================================
 # evaluate loaded model on test set
 MISTAKES_SAVE_PATH = "./visualizations/" + SESSION_NAME + "_mistakes.csv"
-test_accuracy, test_loss, cm = evaluate(model, (MODEL_NAME == "inception"), loss_func, test_dataloader, True, MISTAKES_SAVE_PATH,device)
+test_accuracy, test_loss, cm, output_probabilities, corresponding_ground_truths = evaluate(model, (MODEL_NAME == "inception"), loss_func, test_dataloader, True, MISTAKES_SAVE_PATH,device)
```

Comment:
hmm this function has hella outputs, maybe we should condense them into a tuple or dict or something?